ktor-server/ktor-server-servlet/jvm/src/io/ktor/server/servlet/Attributes.kt

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2014-2020 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.server.servlet
+
+import io.ktor.application.*
+import io.ktor.server.engine.*
+import io.ktor.util.*
+import javax.servlet.*
+
+/**
+ * Provides javax.servlet request attributes or fail it the underlying engine is not
+ * servlet-backed.
+ */
+public val ApplicationCall.servletRequestAttributes: Map<String, Any>
```

Comment:
Should we put it to the request directly?  like `call.request.servletAttributes`

Reply:
Yes, it's possible. Actually, it is what the original PR proposes.